src/videoserver/lib/utils.py

```diff
@@ -99,6 +99,15 @@ def _handle_doc(doc):
                     _external=True
                 )
 
+            if doc['processing']['thumbnail_preview']:
```

Comment:
Guys, I still think that this is redundant..
If you want to keep it: 
```
            if doc['thumbnails']['preview'] or doc['processing']['thumbnail_preview']:
                doc['thumbnails']['preview']['url'] = url_for(
                    'projects.get_raw_preview_thumbnail',
                    project_id=doc['_id'],
                    _external=True
                )
```
To avoid dupllication with the code above.